app/main/lib/image_similarity.py

```diff
@@ -32,6 +32,8 @@ def save(image):
     existing = db.session.query(ImageModel).filter(ImageModel.url==image.url).one()
     existing.context.append(image.context)
     flag_modified(existing, 'context')
+    existing.pdq.append(image.pdq)
+    flag_modified(existing, 'pdq')
```

Comment:
Let's remove these lines and create a different function specifically for updating the pdq hash